packages/yoroi-extension/app/connector/components/signin/cardano/UtxoDetails.js

```diff
@@ -106,6 +106,12 @@ class CardanoUtxoDetails extends Component<Props> {
     addressIndex: number,
     transform?: BigNumber => BigNumber,
   |}) => Node = request => {
+    if (Boolean(request.address)) return null;
+
+    const addressValue = request.address.value;
+
+    if (addressValue === undefined) return null;
```

Comment:
Doesn't seem like having a `null` would be better than undefined in this case, so better to check for both

```suggestion
    if (addressValue == null) return null;
```